test/e2e/vsphere_test.go

```diff
@@ -3599,6 +3599,58 @@ func TestVSphereKubernetes128UbuntuTo129InPlaceUpgrade_3CP_3Worker(t *testing.T)
 	)
 }
 
+func TestVSphereKubernetes125UbuntuTo128InPlaceUpgrade(t *testing.T) {
+	var clusterOpts []framework.ClusterE2ETestOpt
```

Comment:
nit: can we call these kube126clusterOpts, kube127clusterOpts and so on.?

Reply:
Updated